ROOTFS/media/rpi/sage-utils/dhcp-pxe/nfs/etc/installK3S.sh

```diff
@@ -0,0 +1,864 @@
+#!/bin/sh
```

Comment:
See my comment in `release.sh` about downloading this file at build-time instead of committing a version of the file into our code repository.